### DISCUSSION TITLE
Open files calculation for an application/pod

### DISCUSSION THREAD

**Discussion by dheeruymv** [grafana/grafana#52993] [General]
Grafana version: 6.x
OS : RHEL V7

Trying to get open files count for the app’s/pods running on kubernetes platform.

using a metric “Process_files_open_files”

The above metric provides a count but the count is not matching when we do “lsof -p |wc -l ”

Both should ideally match.

Can anyone please let me know that how to get the open files of a pod in kubernetes environment?

If there is no available metric, can you please know any custom methods also to get it?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
